content/v2.0/reference/syntax/annotated-csv/extended.md

```diff
@@ -0,0 +1,360 @@
+---
+title: Extended annotated CSV
+description: >
+  Use the [`influx write` command](/v2.0/reference/cli/influx/write/) to write CSV data
+  to InfluxDB. Include annotations with the CSV data determine how the data translates
+  into [line protocol](/v2.0/reference/syntax/line-protocol/).
+menu:
+  v2_0_ref:
+    name: Extended annotated CSV
+    parent: Annotated CSV
+weight: 201
+v2.0/tags: [csv, syntax, write]
+related:
+  - /v2.0/write-data/csv/
+  - /v2.0/reference/cli/influx/write/
+  - /v2.0/reference/syntax/line-protocol/
+  - /v2.0/reference/syntax/annotated-csv/
+---
+
+**Extended annotated CSV** provides additional annotations and options that specify
+how CSV data should be converted to [line protocol](/v2.0/reference/syntax/line-protocol/)
+and written to InfluxDB.
+InfluxDB uses the [`csv2lp` library](https://github.com/influxdata/influxdb/tree/master/pkg/csv2lp)
+to convert CSV into line protocol.
+Extended annotated CSV supports all [Annotated CSV](/v2.0/reference/syntax/annotated-csv/)
+annotations.
+
+{{% warn %}}
+The Flux [`csv.from` function](/v2.0/reference/flux/stdlib/csv/from/) only supports
+**Annotated CSV**, not **Extended annotated CSV**.
+{{% /warn %}}
+
+To write data to InfluxDB, line protocol must include the following:
+
+- [measurement](/v2.0/reference/syntax/line-protocol/#measurement)
+- [field set](/v2.0/reference/syntax/line-protocol/#field-set)
+- [timestamp](/v2.0/reference/syntax/line-protocol/#timestamp) _(Optional but recommended)_
+- [tag set](/v2.0/reference/syntax/line-protocol/#tag-set) _(Optional)_
+
+Extended annotated CSV annotations identify which element of line protocol each
```

Comment:
Could we get away with saying "Extended CSV annotations identify"? ...to avoid both annotated/annotations

Reply:
...the element of line protocol a column represents?